src/main/resources/db/migration/V3__modify_user_table.sql

```diff
@@ -1,3 +1,2 @@
 ALTER TABLE `user`
-  ADD CONSTRAINT `email_unique` UNIQUE (`email`),
```

Comment:
Why are you changing existing migrations? We discussed it's not allowed 99% of time in real world scenarios

Reply:
yes, I know that, but unfortunately, I did not find another solution.
the problem was that the username column was already dropped from a previous migration, and that 3rd migration was dropping something that was not there causing an error.
That's why I modified that, and I did a flyway repair

But if you have an alternative, I will revert the change.

Reply:
Ok, let's merge it then. As an exercise, you can provide a description how you would fix a problem like that in a production environment. ;)